X/XicTools/bundled/patches/malloc-2.8.6.c.patch

```diff
@@ -0,0 +1,11 @@
+--- ../xictools-master/xt_base/malloc/malloc-2.8.6.c	2024-02-18 20:19:53.000000000 -0500
++++ xt_base/malloc/malloc-2.8.6.c	2024-02-23 08:57:12.595927930 -0500
+@@ -763,7 +763,7 @@
+ /* #define HAVE_USR_INCLUDE_MALLOC_H */
+ 
+ #ifdef HAVE_USR_INCLUDE_MALLOC_H
+-#include "/usr/include/malloc.h"
```

Comment:
Oh wow, this should really fixed upstream, hardcoding the full absolute path of a header file is nonsense.